### PR TITLE
[DOCS-DE] Interactive tutorial

### PR DESCRIPTION
**Pull request by bene2k1**
German translation of the interactive tutorial

### CODE REVIEW

File: content/de/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                    <p><i>Wenn ein Deployment öffentlich verfügbar gemacht wird, wird der Verkehr während des Updates nur auf verfügbare Pods verteilt. </i></p>
+                </div>
+            </div>
+        </div>
+
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p> In the following interactive tutorial, we'll update our application to a new version, and also perform a rollback.</p>
```
Comment (rlenferink) [inline content/de/docs/tutorials/kubernetes-basics/update/update-intro.html]: This sentence is still in English.

**Comment by rlenferink** [comment]
ping @bene2k1

**Comment by bene2k1** [comment]
@rlenferink Done ;-)

**Pull request merged**